Automattic/wp-calypso
# SEO GB Extension: Add

Requires https://github.com/Automattic/jetpack/pull/11333

#### Changes proposed in this Pull Request

Add a Gutenberg extension to include an SEO description panel in the sidebar.

![image](https://user-images.githubusercontent.com/96308/50903747-e41a1600-141e-11e9-990c-f66418e30630.png)

For comparison, this is what the legacy SEO panel in the Calypso Classic Editor looks like:

![image](https://user-images.githubusercontent.com/96308/52673264-7a5ec180-2f20-11e9-8653-187bea6b5706.png)

#### Testing instructions

* Build blocks:
  ```
  npm ci
  npx lerna bootstrap --concurrency=2 --scope='@automattic/jetpack-blocks'
  npx lerna run build --stream --scope='@automattic/jetpack-blocks' -- -- --output-path /path/to/jetpack/_inc/blocks/
  ```
   Remember to change output path to correct.
* Apply this branch and Jetpack's `add/seo-post-meta` branch to your local environment
* Open the Jetpack sidebar or hit "publish" on new post/page
* find the 'SEO Description' panel, enter some text into the textfield.
* Save draft/publish
* Verify that the SEO string is persisted.

Fixes #27692
